Runtest.md

```diff
@@ -0,0 +1,76 @@
+# How to run the test against a) Local sandbox and b) Remote CDAP instance. 
+
+## Local sandbox: 
+
+Provide the local sandbox URL in ConnectionParameters.properties file. 
+
+In the ConnectionParameters.properties file, provide the local sandbox URL. The path of the property file is 
+cdap-e2e-tests>src>main>resource>ConnectionParameters.properties
+
+Property name and its value is CDFURL=http://localhost:11011/pipelines/ns/default/studio
+All the configuration details need to be present in this file.
+
+**Configuration Details required for Setup:**
+
+The cdap-e2e is a framework and google-cloud/src/e2e-test is a plugin specific repo, which contains a plugin specific test and step definition.
+* cdap-e2e-tests
+* google-cloud
+
+_Clone the cdap-e2e-tests framework of develop branch, which contains the common helper classes:_
+
+1. Open IntelliJ IDEA tool.
+2. From the File menu select New and choose Project From Version Control option.
+3. Provide the Github URL in the URL and click on Clone to clone the project.
+
+_URL: https://github.com/cdapio/cdap-e2e-tests_
+
+4. Provide the required details in the ConnectionParameters.properties file that is common for all the connectors. 
+
+_Please find the list of connectors along with their specific configuration details:-_
+
+1. GCS: When we have GCS as source, we have to provide the path of the file located in the GCS bucket which already exists. All the different file formats paths used for testing are configured here, it also includes the GCP credentials. 
+
+GCSbucket- Provide the bucket path
+For example, gs://<bucket>/path/to/directory/
+			 project-id= XXXXX
+
+### Add Plugin specific Actions and Locators:
```

Comment:
This section should be moved to the doc that describes how to create new tests

Reply:
Done.